about.html

```diff
@@ -41,14 +44,17 @@
             <div class="main-navigation">
                 <div class="logo">
                     <a href="index.html">
-                        <img src="">
+                        <img alt="Robust-Events" src="img/logo.png" class="main-logo">
                     </a>
                 </div>
 
                 <div class="menu-icon"></div>
 
                 <div class="navigation">
                     <ul class="menu">
+                        <li class="menu-item">
+                            <a href="index.html" class="menu-link home-link">Home</a>
+                        </li>
                         <li class="menu-item">
                             <a href="about.html" class="menu-link">About</a>
                         </li>
```

Comment:
- [ ] Please, consider that It's possible to navigate between the pages, there's a link from the `Home` page to the `About` page and **the opposite too**. You have a logo for the link to the Home page, but you should also implement the `Home` link on the About page, as required.